scripts/globals/helm.lua

```diff
@@ -1392,7 +1392,7 @@ local helmInfo =
 -- colored rocks. do not change this order!
 -----------------------------------
 
-local rocks = { 769, 771, 770, 772, 773, 774, 776, 775 }
+local rocks = { 769, 773, 772, 771, 774, 770, 776, 775 }
```

Comment:
Two thoughts here:
1. It concerns me that there is a comment above that says `do not change this order!`. If is truly ok for us to change then we should remove that comment.
2. If we do want these to match the elements then we should do so explicitly. Using the constant values makes things much more readable too!

```lua
-- At top of file
require("scripts/globals/items")

local rocks =
{
    [xi.magic.element.FIRE] = xi.items.RED_ROCK,
    [xi.magic.element.ICE] = xi.items.TRANSLUCENT_ROCK,
    [xi.magic.element.WIND] = xi.items.GREEN_ROCK,
    [xi.magic.element.EARTH] = xi.items.YELLOW_ROCK,
    [xi.magic.element.THUNDER] = xi.items.PURPLE_ROCK,
    [xi.magic.element.WATER] = xi.items.BLUE_ROCK,
    [xi.magic.element.LIGHT] = xi.items.WHITE_ROCK,
    [xi.magic.element.DARK] = xi.items.BLACK_ROCK,
}
```

Reply:
+1 for concern about the comment: do not change this order!

I remember there being a big refactor to try and bring weather's and elements and all that sort of stuff in line a long time ago and these things can be very sneaky if handled incorrectly. Proceed with caution!

Reply:
Thanks for looking into that, sounds good 🙏👍

Reply:
Updated per recommendations.